GVFS/GVFS.Platform.Windows/WindowsFileBasedLock.cs

```diff
@@ -0,0 +1,157 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Tracing;
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Text;
+
+namespace GVFS.Platform.Windows
+{
+    public class WindowsFileBasedLock : FileBasedLock

```

Comment:
I'm guessing there are no meaningful changes in this code compared to the old `FileBasedLock`?

Reply:
There were a lot of changes to this file to remove dead code and clean up the interface.

If you'd like to see the changes you can check out the first commit:

https://github.com/Microsoft/VFSForGit/pull/213/commits/2ab1b2b6d7275e0a7af42a0bf8e2bf158fb52d7f

I did not perform the file rename in that commit to make it easier to see the diffs in this file (see GVFS/GVFS.Common/FileBasedLock.cs in the above commit)

Reply:
(Note that these changes pre-dated the creation of the abstract FileBasedLock, and so the private members were not yet moved out)